src/bitcoin-cli.cpp

```diff
@@ -506,7 +510,10 @@ class NetinfoRequestHandler : public BaseRequestHandler
                 const bool is_addr_relay_enabled{peer["addr_relay_enabled"].isNull() ? false : peer["addr_relay_enabled"].get_bool()};
                 const bool is_bip152_hb_from{peer["bip152_hb_from"].get_bool()};
                 const bool is_bip152_hb_to{peer["bip152_hb_to"].get_bool()};
-                m_peers.push_back({addr, sub_version, conn_type, network, age, min_ping, ping, addr_processed, addr_rate_limited, last_blck, last_recv, last_send, last_trxn, peer_id, mapped_as, version, is_addr_relay_enabled, is_bip152_hb_from, is_bip152_hb_to, is_outbound, is_tx_relay});
+                std::string name = network;
+                if (NETWORK_SHORTNAMES.find(name) != NETWORK_SHORTNAMES.end())
+                    name = NETWORK_SHORTNAMES.find(name)->second;
```

Comment:
Needs braces at least, but it would be nice to avoid `find`ing twice too.

Reply:
thanks fixed in bf69a13677680dbf2065e18e3e5c7882ec3aab33